flowzone.yml

```diff
@@ -2980,6 +3045,161 @@ jobs:
           secrets: ${{ toJSON(secrets) }}
           variables: ${{ toJSON(vars) }}
 
+  ###################################################
+  # AWS/CloudFormation
+  ###################################################
+
+  cloudformation_test:
+    name: Test CloudFormation
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    strategy:
+      fail-fast: true
+      matrix:
+        stack: ${{ fromJSON(needs.is_cloudformation.outputs.stacks) }}
+    timeout-minutes: ${{ fromJSON(inputs.jobs_timeout_minutes) }}
+    needs:
+      - is_pr_open
+      - is_cloudformation
+    if: |
+      !failure() && !cancelled() &&
+      needs.is_pr_open.result == 'success' &&
+      needs.is_cloudformation.outputs.cloudformation == 'true'
+
+    <<: *customWorkingDirectory
+
+    outputs:
+      change_set_command: ${{ steps.change_set.outputs.command }}
+
+    steps:
+      - *downloadSourceArtifact
+      - *extractSourceArtifact
+      - *configureAWSCredentials
```

Comment:
`configureAWSCredentials` will be skipped on external PRs, do we want the other steps to continue?